component-classes/index.js

```diff
@@ -407,14 +407,13 @@ export const select = {
 
 export const label = {
   label: 'antialiased block relative text-s font-bold pb-4 cursor-pointer s-text',
-  labelInvalid: 's-text-negative',
-  optional: 'pl-8 font-normal text-s s-text',
+  optional: 'pl-8 font-normal text-s s-text-subtle',
```

Comment:
Hmm.... Are we "allowed" to remove these? 🤔  As this would mean that any uses of these classes would break.

Reply:
For reference: `transparentBg` in line 352 is marked for removal in v2.

Reply:
The reason I removed `labelInvalid` key was that I haven't found any apps that would use `label.labelInvalid` classes (like maybe was the case with `modal.transparentBg` in line 352). So in this case it wouldn't break for anyone updating css package. However, I agree that we should be consistent and rather keep this property + add a deprecation comment like you did in line 352. 

On a side note, we'll probably end up with many similar situations when rewriting the `input` classes because they are used in [Dave's](https://github.schibsted.io/finn/pf-speedcalculator-widget/blob/24b50eeed521810b6aa1b59f306655d070b5919d/client/Input.svelte) and [Vegard's app](https://github.schibsted.io/finn/search-newfrontier-podlet/blob/2279c43f0a108b32252004436fdcfcd81a52178d/uno.config.js#L1). So any changes in the class strings would break their implementations of text fields... We could potentially create completely new properties though, and not touch the old ones, just for the sake of backwards compatibility.

Reply:
Sounds good 🙂 👍

Reply:
Fixed in [b0b836c](https://github.com/warp-ds/css/pull/180/commits/b0b836c35437c3169c70f60cc886b7ef1102bf05) and [1287dd6](https://github.com/warp-ds/css/pull/180/commits/1287dd6fb568d4fd207eb287d31af0aed20a422c